modal/config.py

```diff
@@ -135,6 +136,32 @@ def config_set_active_profile(env: str) -> None:
     _write_user_config(_user_config)
 
 
+def _check_config() -> None:
+    num_profiles = len(_user_config)
+    num_active = sum(v.get("active", False) for v in _user_config.values())
+    if num_active > 1:
+        raise InvalidError(
+            "More than one Modal profile is active. "
+            "Please fix with `modal profile activate` or by editing your Modal config file."
```

Comment:
nit: could add `{user_config_path}` here too to let the user know where the config file is

Reply:
Ooo nice idea!